docs/seps/sep-0005.rst

```diff
@@ -11,67 +11,94 @@ SEP 5 — SDyPy Unified timeseries
+
+
+.. code-block::
+
+  {
+        "data": np.array([1,2,3,4]),                      # Array of data shaped (n,) (or (m, n) for `m` channels) and `n` time-samples.
+        "time": np.array([1,3,4,7]),                      # Time-vector in seconds. Optional for equally spaced data.
+        "fs": 125,                                        # Sampling frequency in Hz. Optional if `time` specified.
+        "name": "channel_id_1",                           # Channel name (or list of `m` channel names if `data` is of shape (m, n)).
+        "start_timestamp": "2023-06-29T15:00:00.000000",  # Optional: timestamp in absolute time. Default format is ISO 8601.
+        "start_timestamp_format": "%d/%m/%y %H:%M:%S.%f"  # Optional: if custom format is used in `start_timestamp`, specify it here.
+        "unit_str": "m/s²",                               # Optional: unit string (or list of `m` unit strings if `data` is of shape (m, n)).
+        "unit_tex": "m/s$^2$",                            # Optional: LaTeX interpretation of `unit_str`.
+        "...": '...'                                      # Optional: additional data.
+  }
+
+
+To represent multiple timeseries with potentially different time-sampling parameters, a `list` of timeseries-object dictionaries is perferred:
 
 
 .. code-block::
 
   [
-    {
-         "data": np.Array([1,2,3,4]) ,                     # 1D array of data
-         "fs": 125,                                        # Sampling frequency in Hz
-         "time": np.Array([1,3,4,7]),                      # Sampling freq is required for equally spaced data, while `time` for non-equally
-         "name": "channel_id_1",                           # channel name
-         "start_timestamp": "2023-06-29T15:00:00.000000",  # Optional: Timestamp in absolute time default format is ISO 8601 ()
-         "start_timestamp_format": "%d/%m/%y %H:%M:%S.%f"  # Optional: if custom format is used in `start_timestamp`, specify it here
-         "unit_str": "m/s²",                               # Optional: Unit string
-         "unit_tex": "m/s$^2$",                            # Optional: Unit latex string
-         "whatever": whatever                              # Optional
-    },{...}
+    { # timeseries 1
+          "data": ...,
+          "time": ...,
+          "fs": ..,                                       
+          "name": ..., 
+    },
+    
+    { # timeseries 2
+    ... 
+    },
   ]
 
+
 Compulsory fields
 .................
 
-* ``data`` (np.Array): 1D or 2D array of data, 1D arrays are prefered but not enforced to allow for e.g. image-based 
-* ``name`` (str): Unique name of the channel (uniqueness within the list is compulsory)
+* ``data`` (np.array): 1D (single channel) or 2D (multiple channels with equal time sampling) array of timeseries data. 
+* ``name`` (str): Unique name of the channel (uniqueness within the list multiple timeseries is compulsory).
```

Comment:
we need to be careful to avoid confusion, maybe we shouldn't use  the terms channels and timeseries together, perhaps makes it confusing.

Reply:
In the next iteration of the document, I've tried to make this less confusing, but I struggle to find better ways of expressing this distinction. What I went with is still: we have a single timeseries object, that can represent multiple measured channels.

Any better suggestions?